builds/e2e/templates/e2e-run.yaml

```diff
@@ -18,22 +20,31 @@ steps:
       $filter += '&Category!=UnstableOnArm'
     }
     
-    #filter out tests not usable on nested edge
-    if ('$(nestededge)' -eq 'true')
+    if ($test_type -eq 'nestededge_mqtt')
     {
       $filter += '&Category!=SingleNodeOnly'
     
       #We are disabling some additional tests. See  PBI #9171870. Once this is fixed we should run what's directly below.
       #$filter += '&FullyQualifiedName!~Provisioning&FullyQualifiedName!~PlugAndPlay&FullyQualifiedName!~PriorityQueueModuleToHubMessages&FullyQualifiedName!~SasOutOfScope'
       $filter += '&FullyQualifiedName!~Provisioning&FullyQualifiedName!~PlugAndPlay&FullyQualifiedName!~PriorityQueueModuleToHubMessages&FullyQualifiedName!~SasOutOfScope&FullyQualifiedName!~X509ManualProvision&FullyQualifiedName!~AuthorizationPolicyUpdateTest&FullyQualifiedName!~AuthorizationPolicyExplicitPolicyTest'
     }
-    else
+    elseif ($test_type -eq 'nestededge_amqp')
+    {
+      $filter += '&Category!=SingleNodeOnly'
+      $filter += '&FullyQualifiedName!~Provisioning&FullyQualifiedName!~PriorityQueueModuleToHubMessages&FullyQualifiedName!~SasOutOfScope&FullyQualifiedName!~X509ManualProvision&FullyQualifiedName!~AuthorizationPolicyUpdateTest&FullyQualifiedName!~AuthorizationPolicyExplicitPolicyTest'
+    }
+    elseif ($test_type -eq 'nestededge_isa95')
     {
-        $filter += '&Category!=NestedEdgeOnly'
+      $filter = 'Category=nestededge_isa95'
     }
+    else
+    {
+      $filter += '&Category!=NestedEdgeOnly' 
+    } 
 
     sudo --preserve-env dotnet test $testFile --logger:trx --testcasefilter:$filter  
-  displayName: Run tests
+  displayName: Run tests ${{ parameters.test_type }}
+  continueOnError: true
```

Comment:
@huguesBouvier can I ask why this change? it makes it harder to re-run failed tests in the pipeline. If you want some other steps to still execute even if tests are failed, you can use `condition: always` instead.